stable/insights-agent/templates/release-watcher/_job.yaml

```diff
@@ -0,0 +1,39 @@
+{{- define "relasewatcher.job.spec" -}}
+restartPolicy: Never
+serviceAccountName: {{ include "insights-agent.fullname" . }}
+volumes:
+- name: output
+  emptyDir: {}
+containers:
+- name: relase-watcher-audit
+  image: "{{ .Values.relasewatcher.image.repository }}:{{ .Values.relasewatcher.image.tag }}"
+  imagePullPolicy: Always
+  args:
+    - cluster
+    - -u
+    - https://charts.fairwinds.com/stable
+    - -u
+    - https://charts.fairwinds.com/incubator
+    - -u
+    - https://kubernetes-charts.storage.googleapis.com
+    - -u
+    - https://kubernetes-charts-incubator.storage.googleapis.com
+    - -o
+    - /output/release-watcher.json
```

Comment:
`file` should be passed to the `-o` flag.  To specify a path, you can use the `--file-path` flag.  Ex:
```
- -o
- file
- --file-path
- /output/release-watcher.json
```